upgrade/upgrade-3.6.0.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen PrestaShop plugin
+ *
+ * @author Adyen BV <support@adyen.com>
+ * @copyright (c) 2021 Adyen B.V.
+ * @license https://opensource.org/licenses/MIT MIT license
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+// This file declares a function and checks if PrestaShop is loaded to follow
+// PrestaShop's good practices, which breaks a PSR1 element.
+//phpcs:disable PSR1.Files.SideEffects
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+/**
+ * This function is automatically called on version upgrades.
+ *
+ * @param AdyenOfficial $module
+ *
+ * @return bool
+ */
+function upgrade_module_3_6_0(AdyenOfficial $module)
+{
+    return $module->copyEmailTemplates() && install_new_tabs($module);
+}
+
+/**
+ * @param AdyenOfficial $module
+ *
+ * @return bool
+ */
+function install_new_tabs(AdyenOfficial $module)
```

Comment:
Where are these tabs installed in the adyenofficial.php? In case someone installs it using the latest version the upgrade script won't run. Recommended to create this function in the adyenofficial.php and then just use that function in the upgrade script so you don't have to repeat the code.

Reply:
These tabs are installed using the `installTabs` function in `adyenofficial.php`. I could have create a function and called it in the upgrade script, but I didn't because in the upgrade script, before adding a new tab I added a check to ensure that the tab is not already added.

This is to ensure the scenario where a merchant installed v3.5.0 and hence, he already has the tabs installed. Going forward (v3.6.0+) these checks will not be required, which is why they are only done in the upgrade script.

Reply:
perfect, I just couldn't find the installation script in this PR and assumed if the upgrade script includes them then this is the version when these tabs were introduced in the plugin as well.

Reply:
Yeah, not including the upgrade script on version 3.5.0 was a mistake on my part, which should be fixed by the next version